app/models/work_package/pdf_export/overview_table.rb

```diff
@@ -143,17 +143,17 @@ def build_nr_cell(work_package, id_wp_meta_map)
   end
 
   def build_subject_cell(content, work_package, id_wp_meta_map)
-    opts = overview_table_cell_padding_style
-    padding_left = opts[:padding_left]
+    opts = styles.overview_table_cell
+    padding_left = opts[:padding_left] || 0
```

Comment:
:memo: When you want to provide a default value for a hash lookup you can also use 

```suggestion
    padding_left = opts.fetch(:padding_left, 0)
```

To some that looks nicer. No need to change, just pointing it out